crates/router/src/compatibility/stripe/customers.rs

```diff
@@ -110,8 +110,9 @@ pub async fn customer_update(
     };
 
     let customer_id = path.into_inner();
-    let mut cust_update_req: customer_types::CustomerRequest = payload.into();
+    let mut cust_update_req: customer_types::CustomerUpdateRequest = payload.into();
     cust_update_req.customer_id = Some(customer_id);
+    let customer_update_id = customer_types::UpdateCustomerId::new("temp_global_id".to_string());
```

Comment:
This can be unit struct right?

Reply:
no, we are not able to pass it as unit struct, it throws error. also the struct i have made is a tuple struct!